### ISSUE TITLE
pakbak script throws error if archive size < keep_number

### ISSUE THREAD

**Issue by svettwer** [gs93/pakbak#4]
Hi!

I found that the pakbak script terminates with error code 123, if the archive contains few files as specified in keep_number within /etc/pakbak.conf.

My config:
```
[sven@arch1337]: ~>$ cat /etc/pakbak.conf 
# Backup the database to this folder
target_folder=/home/sven/Dokumente/backups/pakbak

# Define how long backups should be kept
# Can be a number of days or empty to disable
keep_days=5

# Define how many backups should be kept
# If more backup are found, the oldest are deleted
# Can be a number of file or empty to disable
keep_number=5
```

Example showing the symptoms:
```
[sven@arch1337]: ~>$ source /etc/pakbak.conf 
[sven@arch1337]: ~>$ ls -l ${target_folder}
insgesamt 49744
-rw-r--r-- 1 sven users 12732628 15. Okt 17:35 pakbak-2017-10-15-173528.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:36 pakbak-2017-10-15-173632.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:36 pakbak-2017-10-15-173650.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:37 pakbak-2017-10-15-173728.tar.xz
[sven@arch1337]: ~>$ sh /usr/lib/systemd/scripts/pakbak 
tar: Entferne führende „/“ von Elementnamen
rm: fehlender Operand
„rm --help“ liefert weitere Informationen.
[sven@arch1337]: ~>$ echo $?
123
[sven@arch1337]: ~>$ ls -l ${target_folder}
insgesamt 62180
-rw-r--r-- 1 sven users 12732628 15. Okt 17:35 pakbak-2017-10-15-173528.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:36 pakbak-2017-10-15-173632.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:36 pakbak-2017-10-15-173650.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:37 pakbak-2017-10-15-173728.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:51 pakbak-2017-10-15-175128.tar.xz
[sven@arch1337]: ~>$ sh /usr/lib/systemd/scripts/pakbak 
tar: Entferne führende „/“ von Elementnamen
[sven@arch1337]: ~>$ echo $?
0
[sven@arch1337]: ~>$ ls -l ${target_folder}
insgesamt 49744
-rw-r--r-- 1 sven users 12732628 15. Okt 17:36 pakbak-2017-10-15-173632.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:36 pakbak-2017-10-15-173650.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:37 pakbak-2017-10-15-173728.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:51 pakbak-2017-10-15-175128.tar.xz
-rw-r--r-- 1 sven users 12732628 15. Okt 17:55 pakbak-2017-10-15-175511.tar.xz
```

BR,
Sven

**Comment by FMAOuroboros** [comment]
Hi again!
I've created a local branch solving this issue but I cant push it nor can I create a branch within the web interface. =/

BR,
Sven

**Comment by gs93** [comment]
Was fixed by #5
